examples/osp-flatcar-aws.yaml

```diff
@@ -0,0 +1,511 @@
+        Type=oneshot
+        RemainAfterExit=true
+        EnvironmentFile=-/etc/environment
+        ExecStart=/opt/bin/setup
+
+  files:
+    - path: /etc/systemd/journald.conf.d/max_disk_use.conf
+      content:
+        inline:
+          encoding: none
+          data: |
+            [Journal]
+            SystemMaxUse=5G
+
+    - path: /opt/load-kernel-modules.sh
+      permissions: 0755
+      content:
+        inline:
+          encoding: none
+          data: |
+            #!/usr/bin/env bash
+            set -euo pipefail
+
+            modprobe ip_vs
+            modprobe ip_vs_rr
+            modprobe ip_vs_wrr
+            modprobe ip_vs_sh
+
+            if modinfo nf_conntrack_ipv4 &> /dev/null; then
+              modprobe nf_conntrack_ipv4
+            else
+              modprobe nf_conntrack
+            fi
+
+    - path: /etc/sysctl.d/k8s.conf
+      content:
+        inline:
+          encoding: none
+          data: |
+            net.bridge.bridge-nf-call-ip6tables = 1
+            net.bridge.bridge-nf-call-iptables = 1
+            kernel.panic_on_oops = 1
+            kernel.panic = 10
+            net.ipv4.ip_forward = 1
+            vm.overcommit_memory = 1
+            fs.inotify.max_user_watches = 1048576
+
+    - path: /opt/bin/setup_net_env.sh
+      permissions: 0755
+      content:
+        inline:
+          encoding: none
+          data: |
+            #!/usr/bin/env bash
+            echodate() {
+              echo "[$(date -Is)]" "$@"
+            }
+
+            # get the default interface IP address
+            DEFAULT_IFC_IP=$(ip -o  route get 1 | grep -oP "src \K\S+")
+
+            if [ -z "${DEFAULT_IFC_IP}" ]
+            then
+              echodate "Failed to get IP address for the default route interface"
+              exit 1
+            fi
+
+            # get the full hostname
+            FULL_HOSTNAME=$(hostname -f)
+
+            # write the nodeip_env file
+            # we need the line below because flatcar has the same string "coreos" in that file
+            if grep -q coreos /etc/os-release
+            then
+              echo -e "KUBELET_NODE_IP=${DEFAULT_IFC_IP}\nKUBELET_HOSTNAME=${FULL_HOSTNAME}" > /etc/kubernetes/nodeip.conf
+            elif [ ! -d /etc/systemd/system/kubelet.service.d ]
+            then
+              echodate "Can't find kubelet service extras directory"
+              exit 1
+            else
+              echo -e "[Service]\nEnvironment=\"KUBELET_NODE_IP=${DEFAULT_IFC_IP}\"\nEnvironment=\"KUBELET_HOSTNAME=${FULL_HOSTNAME}\"" > /etc/systemd/system/kubelet.service.d/nodeip.conf
+            fi
+
+    - path: /opt/bin/setup
+      permissions: 0755
+      content:
+        inline:
+          encoding: none
+          data: |
+            #!/bin/bash
+            set -xeuo pipefail
+            if systemctl is-active ufw; then systemctl stop ufw; fi
+            systemctl mask ufw
+
+            {{- if .FlatcarConfig.DisableUpdateEngine }}
```

Comment:
What is this `DisableUpdateEngine` ? It should be `DisableAutoUpdate`

Reply:
Fixed